tests/scripts/thread-cert/pktverify/layer_fields.py

```diff
@@ -530,8 +530,7 @@ def __call__(self, v: Union[LayerFieldsContainer, LayerField]):
     'thread_nwd.tlv.server.16': _auto,
     'thread_nwd.tlv.border_router.16': _auto,
     'thread_nwd.tlv.sub_tlvs': _list(_str),
-    'thread_nwd.tlv.prefix.length': _auto,
-    'thread_nwd.tlv.prefix.domain_id': _auto,
+    'thread_nwd.tlv.prefix': _list(_str),
```

Comment:
Can you add a `TODO: support all fields` here and comment out thread_nwd.tlv.prefix.length and thread_nwd.tlv.prefix.domain_id (instead of removing them) ?

Reply:
sure